libslab/libslab-utils.c

```diff
@@ -221,7 +221,11 @@ libslab_mate_desktop_item_open_help (MateDesktopItem *item)
 	if (doc_path) {
 		help_uri = g_strdup_printf ("help:%s", doc_path);
 
+#if GTK_CHECK_VERSION (3, 22, 0)
```

Comment:
I don't see `libslab_mate_desktop_item_open_help` being used anywhere, so it probably can be dropped.